<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>58</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stack">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QCheckBox" name="nameIn">
         <property name="text">
          <string>name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="pullup">
         <property name="text">
          <string>Pull-Up</string>
         </property>
         <property name="class" stdset="0">
          <string>pullup</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>57</width>
           <height>26</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QCheckBox" name="nameOut">
         <property name="text">
          <string>name</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>136</width>
           <height>26</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QSlider" name="slider">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLCDNumber" name="count">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="numDigits">
          <number>5</number>
         </property>
         <property name="digitCount">
          <number>5</number>
         </property>
         <property name="segmentStyle">
          <enum>QLCDNumber::Flat</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QSlider" name="thresholdSlider">
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="thresLabel">
         <property name="toolTip">
          <string>The GREEN LED attached to PD5 will toggle when the counter crosses this</string>
         </property>
         <property name="text">
          <string>PD5â‡†</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>85</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="maxCount">
      <number>10</number>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>    INPUT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>OUTPUT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>     CNTR</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>nameOut</sender>
   <signal>clicked(bool)</signal>
   <receiver>Frame</receiver>
   <slot>setOutputState(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pullup</sender>
   <signal>clicked(bool)</signal>
   <receiver>Frame</receiver>
   <slot>setOutputState(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Frame</receiver>
   <slot>setPage(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>count</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>12</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>thresholdSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Frame</receiver>
   <slot>setThreshold(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setOutputState(bool)</slot>
  <slot>setPage(int)</slot>
  <slot>setThreshold(int)</slot>
 </slots>
</ui>
